src/gsi/hybrid_ensemble_parameters.f90

```diff
@@ -497,6 +491,9 @@ subroutine init_hybrid_ensemble_parameters
   l_both_fv3sar_gfs_ens=.false.
   n_ens_gfs=0 
   n_ens_fv3sar=0
+  r_ensloccov4tim=one
```

Comment:
Here is one example when equality evaluations are used for those options in real numbers.   As mentioned above, that is no supposed to work like that, from my understanding of this issue.